app/controller.js

```diff
@@ -4,12 +4,31 @@ module.exports = store => {
     return store.saveUser(user);
   };
 
+
   const findAllUsers = Users => {
     return store.findAllUsers(Users)
   }
 
+
+  const findSingleUser = userId => {
+    return store.findSingleUser(userId)
+  }
+
+
+  const findUserByIdAndUpdate = userId => {
```

Comment:
You can simply this : 
```
const findUserByIdAndDelete = userId => store.findUserByIdAndDelete(userId)
```
  
```
  const findUserByIdAndDelete = store.findUserByIdAndDelete
```

Probably it's not super important that you know what point free is at the moment , but I think it's going to help you a lot when readers other functional code --> https://lucasmreis.github.io/blog/pointfree-javascript/

Reply:
Discussed this over slack and we're going to hold off on this and maybe come back to it in the future as I'm just starting to get the hang of this now and about to jump into building the frontend in react, and I'm worried about overloading with information.  Pinned this however in the convo and will be coming onto it for sure.

NDR : Be worth checking out https://tech.io/playgrounds/2980/practical-introduction-to-functional-programming-with-js/pure-functions also as recommended